docker-compose-reorg.yml

```diff
@@ -0,0 +1,284 @@
+version: "2.3"
+services:
+  geth-1:
+    image: ethereum/client-go:alltools-stable
+    hostname: geth-1
+    environment:
+      - account=0x6de4b3b9c28e9c3e84c2b2d3a875c947a84de68d
+      - bootnodes=enode://b655cc3e5b72ab9beb8a8536a3c3ae92fbeb79feb1ebd7f95d72be72554ca586428bd48a54eb9c2bcaae455cc674299b6dd3df3c6556a493dfd50070f1a448aa@172.25.0.103:30303
+    entrypoint:
```

Comment:
maybe now is not a good time - or if you think it is, I've once extracted this command outside docker-compose-reorg.yml
take a look here: 
https://github.com/omgnetwork/childchain/blob/master/docker-compose.yml#L39
https://github.com/omgnetwork/childchain/blob/master/docker-compose.yml#L50
https://github.com/omgnetwork/childchain/blob/master/docker/geth/command

Reply:
I extracted this command into a file in snapshot